packages/odyssey-babel-loader/src/index.js

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+module.exports = require("babel-loader").custom(() => ({
+  customOptions({ include, loader = {} }, { source }) {
+    const regexp = buildRegexp(include);
+
+    // eslint-disable-next-line no-unused-vars
+    Array.from(source.matchAll(regexp)).forEach(([_, dependency]) => {
+      const absoluteDependency = this.utils.absolutify(
+        this.context,
+        dependency
+      );
+      this.addDependency(absoluteDependency);
+    });
+    return { loader };
+  },
+}));
+
+function buildRegexp(include) {
+  const normalized = Array.isArray(include)
+    ? include.map(toRegexpString)
+    : ["\\.module\\.s?css"];
+
+  return new RegExp(`from ["'\`](\\S+(?:${normalized.join("|")}))["'\`]`, "gi");
+}
+
+function toRegexpString(include) {
+  if (typeof include === "string") return include;
+  if (include instanceof RegExp) return include.toString().slice(1, -1);
```

Comment:
I feel like it's hacky to remove the leading and trailing forward slash `/` in the string this way, but I understand why we're doing it based on the actual code that uses it.

Would we ever want to `trim()` before slicing or is this safe?

Reply:
thanks for eyes, I don't believe trim will be necessary no, well formed arguments to this API won't involve spaces and I've already written enough defensive code here I don't think we need to cover that edge case